source/localizable/press.ca.html.erb

```diff
@@ -0,0 +1,60 @@
+Decidim és gestionat per l'associació del mateix nom que fa servir el codi, les seves extensions i governança.</p>
+
+<h2>Què fem</h2>
+<p>Som un programari de codi lliure per a ciutats i organitzacions. Som una infraestructura participativa. Treballem per una democràcia més forta i transparent.</p>
+
+<h2>On trobar-nos</h2>
+<p>Canòdrom. Parc de Recerca Creativa<br>
+Carrer de Concepción Arenal, 165, 08027 Barcelona</p>
+
+<h2>Xifres</h2>
+<p>Decidim és utilitzat en més de 100 ciutats, en 20 països del mon. Mira <a href="https://decidim.org/ca/usedby/">Ja fan servir Decidim</a>.</p>
+
+
+<h2>Destacats de premsa</h2>
+
+<ul>
+  <li><a href="https://www.deutschlandfunk.de/wenn-buerger-politik-gestalten-barcelonas-erfolg-mit-der.795.de.html?dram:article_id=471680">Barcelonas Erfolg mit der Plattform Decidim.</a> Deutschlandfunk</li>
+  <li><a href="https://www.lavanguardia.com/local/barcelona/20200219/473656039153/tecnologia-ciudad-y-democracia.html">Tecnología, Ciudad y Democracia.</a> La Vanguardia</li>
+  <li><a href="https://bherria.eus/es/una-plataforma-disenada-para-articular-procesos-democraticos-debe-contar-con-una-gobernanza-democratica/">Una plataforma diseñada para articular procesos democráticos debe contar con una gobernanza democrática.</a> Bherria</li>
+  <li><a href="https://www.ft.com/content/67c52480-b51f-11e9-8cb2-799a3a8cf37b">Smart cities still need a human touch.</a> Financial Times</li>
+  <li><a href="https://www.information-age.com/negative-image-of-smart-cities-123485288/">Why we should dispel the negative image of smart cities.</a> Information Age</li>
+  <li><a href="https://www.lemonde.fr/economie/article/2019/06/07/a-barcelone-un-modele-original-de-commun-numerique_5473033_3234.html">A Barcelone, un modèle original de « commun numérique ».</a> Le Monde</li>
+  <li><a href="https://kontrast.at/barcelona-smart-city-wirtschaft/">In Barcelona schreiben die Bürger das Regierungsprogramm selbst – und machen ihre Stadt sozialer.</a> Kontrast</li>
+  <li><a href="https://www.vice.com/it/article/ev3334/m5s-di-maio-offerta-rousseau-gilet-gialli-proteste-francia-democrazia-diretta">L'offerta del M5S di dare Rousseau ai Gilet Gialli suona più come una minaccia.</a> Vice</li>
+  <li><a href="https://netzpolitik.org/2018/freie-software-als-oeffentliches-gut-und-was-rathaeuser-dafuer-tun-koennen/">Interview: Wie Barcelona eine offene "Smart City" im Dienste des Gemeinwohls plant.</a> Netzpolitik</li>
+  <li><a href="https://www.rbb-online.de/doku/b/besser-geht-immer/besser-geht-immer.html">Besser geht immer. Astrid Frohloff trifft kreative Bürokraten</a> RBB</li>
+</ul>
+
+<h2>FAQs</h2>
+<p>Fes clic <a href="https://decidim.org/ca/faqs/">aquí</a>.</p>
+
+<h2>Recursos de comunicació</h2>
+
+<ul>
+  <li><a href="https://decidim.org/pdf/identity-manual-ca.pdf">Logos</a></li>
```

Comment:
it's better to use relative URLs here like
`   <li><a href="/pdf/identity-manual-ca.pdf">Logos</a></li>`

Reply:
the same on FAQ, triptic, etc.

Reply:
Sorry! Done